kcbq-connector/src/integration-test/java/com/wepay/kafka/connect/bigquery/it/utils/BucketClearer.java

```diff
@@ -50,7 +61,8 @@ public static void main(String[] args) {
 
   private static void usage() {
     System.err.println(
-        "usage: BucketClearer <key_file> <project_name> <bucket_name>"
+        "usage: BucketClearer <key_file> <credentials> <project_name> <bucket_name>\n" +
```

Comment:
I'm not a huge fan of this interface, but I'm willing to live with it because this is mostly just called from gradle anyway. However, I think here we should clarify that if key_file or credentials is empty, it's treated as non-existent. (This would also apply to TableClearer)

Reply:
Thank you, @mtagle, I added this.